indicators/facebook-9z3vzzzj2s.yml

```diff
@@ -0,0 +1,18 @@
+title: Facebook phishing kit displaying a faked post
```

Comment:
I think we should include the unique hash (`Facebook phishing kit 9z3vzzzj2s`) in the title so that people can tell it apart from other Facebook phishing kits that also use a fake post.

Reply:
Good point on both sides. Previously I've been including it in the titles but if it's already in the rule ID (and that's displayed prominently enough) then maybe it doesn't matter

I'm happy to merge this as-is and we can come back if we decide on a stricter style-guide in future